packages/astro/src/core/create-vite.ts

```diff
@@ -134,6 +132,8 @@ export async function createVite(
 			vitePluginSSRManifest(),
 			astroAssetsPlugin({ settings, logger, mode }),
 			astroTransitions(),
+			configAliasVitePlugin({ settings }),
+			astroLoadFallbackPlugin({ fs, root: settings.config.root }),
```

Comment:
Move these two to last as `configAliasVitePlugin` should give way for others to resolve first. And `astroLoadFallbackPlugin` is a fallback, it should always be last. The ordering only affects among other `enforce: 'post'` plugins.